test/extended/cli/mustgather.go

```diff
@@ -179,51 +229,46 @@ var _ = g.Describe("[sig-cli] oc adm must-gather", func() {
 					}
 					o.Expect(text).To(o.HavePrefix(`{"kind":"Event",`))
 					for _, token := range []string{"oauthaccesstokens", "oauthauthorizetokens", tokenName} {
-						o.Expect(text).ToNot(o.ContainSubstring(token))
+						o.Expect(text).NotTo(o.ContainSubstring(token))
 					}
 					readFile = true
 					eventsChecked++
 				}
 				// ignore this error as we usually fail to read the whole GZ file
-				// o.Expect(scanner.Err()).ToNot(o.HaveOccurred())
+				// o.Expect(scanner.Err()).NotTo(o.HaveOccurred())
 				o.Expect(readFile).To(o.BeTrue())
 
 				return nil
 			})
-			o.Expect(err).ToNot(o.HaveOccurred())
+			o.Expect(err).NotTo(o.HaveOccurred())
 
 			// On IBM ROKS, events will not be part of the output, since audit logs do not include
 			// control plane logs.
 			if e2e.TestContext.Provider != ibmcloud.ProviderName {
-				o.Expect(eventsChecked).To(o.BeNumerically(">", 10000))
+				o.Expect(eventsChecked).To(o.BeNumerically(">", 1000))
```

Comment:
I'm lowering this to 1000, 10k seems like a lot, and it might depend when in the tests it gets run. I have a cluster that after half a day has about 7k events. so this 1k is a good minimum to start with.